apps/docs/src/app/core/component-docs/carousel/carousel-docs.module.ts

```diff
@@ -30,6 +31,8 @@ const routes: Routes = [
     imports: [
         RouterModule.forChild(routes),
         SharedDocumentationPageModule,
+        BusyIndicatorModule,
```

Comment:
No need to import this.

Reply:
without import it is giving error "fd-busy-indicator" is not a known element.

Reply:
This is the docs module, I don't see any usages of busy indicator there. Check the dependencies, because it really should not be needed.

Reply:
i was removing "fd-busy-indicator" from lib/carousel.module.ts , my mistake. 
removed from docs now. pushed changes.